### PR TITLE
Update aws-codeguru-reviewer.yml

### PR DESCRIPTION
**Pull request by gurudevtest**
Updated third excluded path so that this config file will suppress at least 1 recommendation. Also removed second path since it doesn't exist in the repo.

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/some-package/java/com/mainpackage/FileSyntaxError.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ * For more information, see the Amazon CodeGuru Reviewer User Guide:
+ * https://docs.aws.amazon.com/codeguru/latest/reviewer-ug/welcome.html.
+ */
+public class FileSyntaxError {
+
+    // {fact rule=file-syntax-error@v1.0 defects=1}
+    public void getObjectContentNoncompliant(AmazonS3 amazonS3Client, String bucketName, String key) throws IOException {
+        final S3Object s3object = amazonS3Client.getObject(bucketName, key);
+        // Noncompliant: the statement is incomplete and is missing ";" at the end.
+        System.out.println(s3object.getObjectMetadata())
```
Comment (leepike) [inline src/some-package/java/com/mainpackage/FileSyntaxError.java:22]: Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error, insert &#34;;&#34; to complete Statement

This file contains syntax errors or language features not currently supported by CodeGuru Reviewer in Java versions greater than Java 8, and can&#39;t be analyzed by all of CodeGuru Reviewer&#39;s detectors.

File: src/some-package/java/com/mainpackage/FileSyntaxError.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
+package com.mainpackage;
+
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.model.S3Object;
+import java.io.IOException;
+import java.io.InputStream;
+import com.amazonaws.util.IOUtils;
+
+/**
+ * Even though this file contains weak crypto issues, CodeGuru Reviewer will not
```
Comment (martinschaef) [inline src/some-package/java/com/mainpackage/FileSyntaxError.java:10]: Does it? Looks like its just a syntax error.

File: src/some-package/java/com/mainpackage/FileSyntaxError.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+ * report any issues in it, because it has been excluded in aws-codeguru-reviewer.yml.
+ * 
+ * For more information, see the Amazon CodeGuru Reviewer User Guide:
+ * https://docs.aws.amazon.com/codeguru/latest/reviewer-ug/welcome.html.
+ */
+public class FileSyntaxError {
+
+    // {fact rule=file-syntax-error@v1.0 defects=1}
```
Comment (martinschaef) [inline src/some-package/java/com/mainpackage/FileSyntaxError.java:18]: Are we using this meta-data in this repo as well, or is it only for the detector library?

File: src/some-package/java/com/mainpackage/FileSyntaxError.java (hunk 2 above)
Comment (gurudevtest) [reply src/some-package/java/com/mainpackage/FileSyntaxError.java:10]: Missed this when copying comment. Will update in new revision

File: src/some-package/java/com/mainpackage/FileSyntaxError.java (hunk 3 above)
Comment (gurudevtest) [reply src/some-package/java/com/mainpackage/FileSyntaxError.java:18]: This metadata is from detector library. Let me remove this

**Review by martinschaef — APPROVED** [review]

**Pull request closed without merging**